lib/generators/pundit/policy/templates/policy.rb

```diff
@@ -1,6 +1,12 @@
 <% module_namespacing do -%>
 class <%= class_name %>Policy < ApplicationPolicy
-  class Scope < Scope
+  # NOTE: Prior to Pundit v2.4.0, the inheritance was declared as
```

Comment:
(I'm assuming v2.4.0 will be the next release?)

Reply:
It depends! Right now there are no strict improvements or new features, so I'd probably make it `2.3.2` if released today. Even "prior releases of pundit" should be sufficient, as hopefully that's enough to make people take an extra look.

Reply:
Could also specify "up to 2.3.1" :D